sycl/include/CL/sycl/aliases.hpp

```diff
@@ -24,11 +24,11 @@ class half;
 } // namespace sycl
 } // namespace cl
 
-#ifdef __SYCL_DEVICE_ONLY__
-using half = _Float16;
-#else
+// FIXME: line below exports 'half' into global namespace, which seems incorrect
```

Comment:
Please add reference to "4.10.2.2 Aliases"

Reply:
Why so? "4.10.2.2 Aliases" is about aliases to vector types, while this particular alias is to scalar type

Reply:
Ok, please add a reference to whatever part of the spec (4.10.1?). So far I see no proofs of this claim.

Reply:
Ah, please disregard my comment. There is already a link to the issue in github.